src/schema/slack/types/custom/mod.ts

```diff
@@ -1,9 +1,12 @@
 import { InteractivityType } from "./interactivity.ts";
 import { UserContextType } from "./user_context.ts";
+import { FormInput } from "./form_input.ts";
 
-const CustomSlackTypes = {
+export const CustomSlackTypes = {
   interactivity: InteractivityType,
   user_context: UserContextType,
 };
 
-export { CustomSlackTypes };
+export const InternalSlackTypes = {
+  form_input_object: FormInput,
+};
```

Comment:
We want this separate so that devs don't get this as typeahead on `Slack.schema.functions` since it's not to be used as a param in their functions/workflows